kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -35,6 +35,12 @@ class KubernetesState(OpenMetricsBaseCheck):
     See https://github.com/kubernetes/kube-state-metrics
     """
 
+    class JobCount:
+        def __init__(self):
+            self.count = 0
+            self.previous_run_max_ts = 0
```

Comment:
When I removed that attribute and its usage (just using `current_run_max_ts`), nothing failed. It looks like the jobs are ordered by timestamp in the prometheus response, is that a coincidence? It would simplify the code quite a bit.

Reply:
They're ordered in the text output, but not in the protobuf output iirc. We expected protobuf to be dropped in the migration from Prometheus to OpenMetrics, but it was added back recently, so we would prefer not to rely on ordering here, in case we need to add protobuf support back as well (the specs haven't fully stabilized yet on the community's side, so it's hard to tell).